test/housekeeping_test.js

```diff
@@ -354,54 +355,235 @@ describe('House Keeper', () => {
     }
   });
 
-  it('should call sweepVolumes exactly once', async() => {
-    houseKeeperMock.expects('_sweepVolumes').exactly(regions.length);
-    
+  it('should call handleVolumeData exactly once', async() => {
```

Comment:
@rstiyer and I talked about the tests for this today. We actually can test statsum writes via taskcluster-lib-monitor via setting `mock: true` when setting up the monitor used in testing. [Line 44](https://github.com/rstiyer/ec2-manager/blob/bc2a1ce7703f95fd6621d4de4f1cfc3f10cd47e4/test/housekeeping_test.js#L44) of this file is where we set up the monitor and then we can access it down in these tests. An example of us doing this is in taskcluster-lib-api [here](https://github.com/taskcluster/taskcluster-lib-api/blob/master/test/responsetimer_test.js#L52-L56) and [here](https://github.com/taskcluster/taskcluster-lib-api/blob/master/test/responsetimer_test.js#L70-L89).

Reply:
Once the tests are updated to do this, the `asserts` in `lib/housekeeping.js` should not be as necessary!

Reply:
Oh wait, dustin already talked about this and figured we can follow up with it later. Good enough for me!